core/operation/src/test/java/uk/gov/gchq/gaffer/operation/impl/GetVariableTest.java

```diff
@@ -37,7 +36,7 @@ public void shouldShallowCloneOperation() {
 
         final GetVariable opClone = op.shallowClone();
 
-        assertNotEquals(op, opClone);
```

Comment:
This test is trying to test that the clone is not the same object as the original object. The test should be updated to reflect this